src/test/java/com/wikia/webdriver/testcases/logintests/LoginTests.java

```diff
@@ -49,4 +49,10 @@ public void Login_004_dropDownStaff() {
     dropDown.logIn(credentials.userNameStaff, credentials.passwordStaff);
     base.verifyUserLoggedIn(credentials.userNameStaff);
   }
+
+  @Test(groups = {"Login_005", "Login"})
+  public void Login_005_japaneseUserLogin() {
+    SpecialUserLoginPageObject specialLogin = new SpecialUserLoginPageObject(driver);
+    specialLogin.loginAndVerify(credentials.userNameJapanese2, credentials.passwordJapanese2, "http://ja.ja-test.wikia.com/");
```

Comment:
What do you think about keeping "http://ja.ja-test.wikia.com/" in class variable also?